internal/engine/interfaces/interface.go

```diff
@@ -131,7 +131,6 @@ type EvalStatusParams struct {
 	Result           *Result
 	Profile          *pb.Profile
 	Rule             *pb.Profile_Rule
-	RuleTypeName     string
```

Comment:
Populating the eval params struct with the rule type name was a pain, and it was only used in one place for evaluation telemetry. I decided instead to pass the rule type name via a different mechanism to the telemetry code.